frontend/src/app/components/pool/pool.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { once } from 'process';
+import { BehaviorSubject, combineLatest, from, merge, Observable } from 'rxjs';
+import { delay, distinctUntilChanged, map, scan, startWith, switchMap, tap } from 'rxjs/operators';
+import { BlockExtended, PoolStat } from 'src/app/interfaces/node-api.interface';
+import { ApiService } from 'src/app/services/api.service';
+import { StateService } from 'src/app/services/state.service';
+
+@Component({
+  selector: 'app-pool',
+  templateUrl: './pool.component.html',
+  styleUrls: ['./pool.component.scss']
+})
```

Comment:
See if we can use `changeDetection: ChangeDetectionStrategy.OnPush` here

Reply:
Fixed in https://github.com/mempool/mempool/pull/1242/commits/1e96c93557dafe2eef5460ceb23e5c1b09f3e741